build.gradle

```diff
@@ -190,15 +197,24 @@ subprojects {
 			}
 		}
 	}
-	signing {
-		sign publishing.publications.mavenJava
-	}
-	publishMavenJavaPublicationToMavenRepository.doFirst {
-		if(OSSRH_URL == null) {
-			throw new GradleException("Please specify the variable 'ossrhUrl' in '~/.gradle/gradle.properties'.")
-		}
-		if(version == "unspecified") {
-			throw new GradleException("Please specify the version to release with '-Pversion=X.Y.Z'.")
+
+	publish.doFirst {
+		if(OSSRH_TARGET == 'public') {
+			signing {
```

Comment:
does this really works? AFAIK `signing` is a configuration think not an execution thing

Reply:
I doubt it as well. That's why I want to test this prior to merging.